Dockerfile

```diff
@@ -0,0 +1,96 @@
+# Base container that is used for both building and running the app
+FROM centos:latest as base
+ENV RUBY_SCL="rh-ruby25"
+ENV NODEJS_SCL="rh-nodejs8"
+
+RUN \
+  echo "tsflags=nodocs" >> /etc/yum.conf && \
+  yum -y upgrade && \
+  yum -y install centos-release-scl epel-release && \
+  yum -y install ${RUBY_SCL} ${NODEJS_SCL}-nodejs \
+     mariadb-libs postgresql-libs \
+    ${RUBY_SCL}-ruby{,gems} ${RUBY_SCL}-rubygem-{rdoc,rake,bundler} nc && \
+  yum clean all && \
+  rm -rf /var/cache/yum/
+
+ENV HOME=/home/foreman
+WORKDIR $HOME
+RUN groupadd -r foreman -f -g 1001 && \
+    useradd -u 1001 -r -g foreman -d $HOME -s /sbin/nologin \
+    -c "Foreman Application User" foreman && \
+    chown -R 1001:1001 $HOME
+
+# Add a script to be executed every time the container starts.
+COPY entrypoint.sh /usr/bin/
+RUN chmod +x /usr/bin/entrypoint.sh
+ENTRYPOINT ["entrypoint.sh"]
+
+# Temp container that download gems/npms and compile assets etc
+FROM base as builder
+
+RUN \
+  yum -y install redhat-rpm-config git \
+    gcc-c++ make bzip2 \
+    libxml2-devel libcurl-devel ${RUBY_SCL}-ruby-devel \
+    mariadb-devel postgresql-devel libsq3-devel && \
+  yum clean all && \
+  rm -rf /var/cache/yum/
+
+ENV RAILS_ENV="production"
+ENV FOREMAN_APIPIE_LANGS="en"
+
+ENV HOME=/home/foreman
+ARG BUNDLER_SKIPPED_GROUPS="mysql test development jenkins openid libvirt journald"
```

Comment:
Why do you skip `mysql` here when your database is in fact MySQL?

Reply:
its the mysql (1) gem, everyone moved to mysql2 :) I guess we can remove it in core as I dont really assume it still works?

Reply:
The `mysql` group was removed in 63b3ed2d1d5e95af8fedb3fab8cdde3357a0eafd (dated in 2013). There is also no jenkins group.